packages/component/src/Styles/StyleSet/SendBoxTextarea.js

```diff
@@ -0,0 +1,50 @@
+export default function createSendBoxTextareaStyle({
+  paddingRegular,
+  primaryFont,
+  sendBoxMaxHeight,
+  sendBoxTextColor,
+}) {
+  return {
+    alignItems: 'center',
+    fontFamily: primaryFont,
+    paddingBottom: paddingRegular,
+    paddingLeft: paddingRegular,
+    paddingRight: paddingRegular,
+    paddingTop: paddingRegular,
+
+    '& > div': {
+      color: sendBoxTextColor,
+      fontFamily: 'inherit',
+      fontSize: 'inherit',
+      height: '100%',
+      maxHeight: sendBoxMaxHeight,
+      position:'relative',
+      width: 'inherit',
+
+      '& > div': {
+        color: 'inherit',
+        fontFamily: 'inherit',
+        fontSize: 'inherit',
+        height: '100%',
+        overflowY: 'scroll',
+        whiteSpace: 'pre-wrap',
+        width: 'inherit',
+      },
+
+      '& > textarea': {
+        border: 0,
+        color: 'inherit',
+        fontFamily: 'inherit',
```

Comment:
Even though font-family is set to inherit, the screencap looks like Arial instead of Calibri. I'm also not great with fonts, could you verify for a sanity check?

Reply:
It looks like the font is inheriting correctly. 

![image](https://user-images.githubusercontent.com/17039790/55996932-ceaac700-5c6d-11e9-9979-a5856c93858e.png)